docs/source/publish_pure_python.rst

```diff
@@ -26,6 +27,11 @@ Choose an alternative image for the runner to use for building and
 testing the source distribution and wheel. By default, this is
 ``ubuntu-latest``.
 
+test_groups
+^^^^^^^^^^^
+
+Comma-separated, PEP 735 dependency groups that should be installed for testing.
+
```

Comment:
Thanks @neutrinoceros, this all looks great! I think we should copy this section into `publish.rst` also, then I think we can get this released! Thanks for all your work on this!

Reply:
nice catch. Done !